pkg/helm/release/manager.go

```diff
@@ -69,6 +69,8 @@ type manager struct {
 	chart            *cpb.Chart
 }
 
+type UpdateOption func(*action.Upgrade) error
```

Comment:
While we're at it, can you make similar changes for `InstallRelease()` and `UninstallRelease()` so we can keep the function signatures consistent?

Reply:
Yes, I added two additional types for install and uninstall options